api/package.json

```diff
@@ -46,8 +43,7 @@
     "morgan": "^1.10.0",
     "passport": "0.5.0",
     "passport-cas": "git+https://github.com/coursetable/passport-cas",
-    "prisma": "5.4.0",
-    "ts-node": "^10.9.1",
+    "prisma": "^5.2.0",
```

Comment:
Check node_modules and see what version this resolves to? `^5.2.0` should match the latest version. I don't know if that's the intention; if so, you should directly use `^5.5.2`.

Reply:
I mean sure. Eventually, it'll always be outdated when 5.5.3 or 5.6.0 comes out - that's why we use the carrot semver right?

Reply:
Indeed. My point is you used a lower range that resolves to a higher version which seems confusing. If this is intended, feel free to not do anything, although I personally bump the lower bound whenever I actually mean to bump it.

Reply:
Gotcha. Yea didn't really mean to bump it in this pq but I've updated the ver anyways.